code-challenges.test.js

```diff
@@ -1,69 +1,316 @@
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with expect statements for each of the variables provided.
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+
+// // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
+// // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// // Expected output: [72, 81, 90, 99, 108]
+
+
+// // b) Create the function that makes the test pass.
 
+const timesThree = numbersArray1.map(value => {
+  return value * 3
+})
 
-// b) Create the function that makes the test pass.
+console.log(timesThree)
 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+//1//
+// // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const num1 = 15
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
+// // Expected output: "-7 is not divisible by three"
+
+//----- MY TEST -------
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num1 is divisible by three with no remainder, if statement is true then log num1 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ 
+      expect(outcome(num1)).toEach("15 is divisible by three")
+    })
+})      
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num2 is divisible by three with no remainder, if statement is true then log num2 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ ""
+      expect(outcome(num2)).toEach("0 is divisible by three")
+    })
+}) 
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num3 is divisible by three with no remainder, if statement is true then log num3 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ 
+      expect(outcome(num3)).toEach("-7 is not divisible by three")
+    })
+})
```

Comment:
You should only need to write one describe and it block per problem, the different tests will be on different expect statements instead, so you will have multiple expect statements.